/**:
  ros__parameters:
    dt: 0.033
    ds: 0.1
    use_perception: false
    debug: 2000
    frame_id: "map"
    valid_ttl: [2]
    n_threads: 8
    max_left_deviation: 10.0
    max_right_deviation: 10.0
    max_yaw_diff: 3.14
    min_speed: 2.0
    road_inc: 2.0
    max_speed_low_deviation: 6.0
    max_speed_high_deviation: 6.0
    speed_inc: 2.0
    max_lat_vel_low_deviation: 2.0
    max_lat_vel_high_deviation: 2.0
    lat_vel_inc: 1.0
    t_period: 5.0
    frenet_dt: 0.1
    boundary_safety_thresh: 1.0
    obj_safety_thresh: 4.0
    detection_radius: 30.0
    follow_desired_gap: 30.0
    kp_gap: 1.0
    kc_prev: 1.0
    kj: 1.0
    kd: 0.5
    klat: 1.0
    klon: 3.0
    max_acc: 10.0
    max_curv: 4.0
    marker_size: 0.01
    marker_lifetime: 0.05
    center_ttl: 3
